dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/EPersonRestRepository.java

```diff
@@ -129,6 +126,9 @@ private EPerson createEPersonFromRestObject(Context context, EPersonRest eperson
             eperson.setEmail(epersonRest.getEmail());
             eperson.setNetid(epersonRest.getNetid());
             if (epersonRest.getPassword() != null) {
+                if (!validatePasswordService.isPasswordValid(epersonRest.getPassword())) {
+                    throw new PasswordNotValidException("The new password to set is not valid");
```

Comment:
This could be changed to `throw new PasswordNotValidException();` as the message is never used.